tests/test_spheropolygon.py

```diff
@@ -36,6 +36,16 @@ def test_2d_verts(square_points):
     ConvexSpheropolygon(square_points, 1)
 
 
+@given(floats(0.1, 1000))
+def radius_getter_setter_tests(r, square_points):
+    """Test getting and setting the radius."""
+    square_points = square_points[:, :2]
+    convexspheropolygon = ConvexSpheropolygon(square_points, r)
+    assert convexspheropolygon.radius == r
+    convexspheropolygon.radius = r + 1
+    assert convexspheropolygon.radius == r + 1
```

Comment:
Testing your error behavior is also a good idea.
```suggestion
    assert convexspheropolygon.radius == r + 1
    with pytest.raises(ValueError):
        convexspheropolygon.radius = 'invalid value'
```